commit a7aacce501ed70e29d17a23369dd62e1e152fae0 (refs/changes/06/5606/1)
Author: Rui Loura <rui@joyent.com>
Date:   2019-02-19T19:11:35+00:00 (8 months ago)
    
    MANTA-4010 sdc-headnode setup_manta_zone.sh script should be rack aware for external network

diff --git a/scripts/setup_manta_zone.sh b/scripts/setup_manta_zone.sh
index aa87926a..aee068af 100755
--- a/scripts/setup_manta_zone.sh
+++ b/scripts/setup_manta_zone.sh
@@ -50,7 +50,7 @@ function add_external_nic {
     {
         \"networks\": [
             {
-                \"uuid\": \"${external_net_uuid}\",
+                \"name\": \"external\",
                 \"primary\": true
             }
         ]
